<ui version="4.0" >
 <class>FontPreview</class>
 <widget class="QDialog" name="FontPreview" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>604</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Fonts Preview</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" colspan="2" >
    <widget class="QLabel" name="searchLabel" >
     <property name="text" >
      <string>&amp;Quick Search:</string>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
     <property name="buddy" >
      <cstring>searchEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="2" colspan="5" >
    <widget class="QLineEdit" name="searchEdit" />
   </item>
   <item row="1" column="0" colspan="7" >
    <widget class="QCheckBox" name="extendedCheckBox" >
     <property name="text" >
      <string>Show Extended Font Informations</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="7" >
    <widget class="FontListView" name="fontList" >
     <property name="alternatingRowColors" >
      <bool>true</bool>
     </property>
     <property name="selectionMode" >
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior" >
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="sortingEnabled" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="7" >
    <widget class="QLabel" name="fontPreview" >
     <property name="minimumSize" >
      <size>
       <width>500</width>
       <height>100</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>16777215</width>
       <height>100</height>
      </size>
     </property>
     <property name="frameShape" >
      <enum>QFrame::Box</enum>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0" >
    <widget class="QLabel" name="sizeLabel" >
     <property name="text" >
      <string>&amp;Font Size:</string>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
     <property name="buddy" >
      <cstring>sizeSpin</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2" >
    <widget class="QSpinBox" name="sizeSpin" >
     <property name="minimum" >
      <number>10</number>
     </property>
     <property name="maximum" >
      <number>72</number>
     </property>
    </widget>
   </item>
   <item row="4" column="3" >
    <widget class="QLabel" name="displayLabel" >
     <property name="text" >
      <string>Text</string>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="4" >
    <widget class="QLineEdit" name="displayEdit" >
     <property name="toolTip" >
      <string>Sample text to display</string>
     </property>
    </widget>
   </item>
   <item row="4" column="5" >
    <widget class="QPushButton" name="displayButton" >
     <property name="toolTip" >
      <string>Set the preview text</string>
     </property>
     <property name="text" >
      <string>Se&amp;t</string>
     </property>
     <property name="shortcut" >
      <string>Alt+T</string>
     </property>
    </widget>
   </item>
   <item row="4" column="6" >
    <widget class="QPushButton" name="resetDisplayButton" >
     <property name="toolTip" >
      <string>Reset the text</string>
     </property>
     <property name="text" >
      <string>&amp;Default</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="7" >
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton" >
       <property name="text" >
        <string>&amp;Close</string>
       </property>
       <property name="shortcut" >
        <string>Alt+C</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <customwidgets>
  <customwidget>
   <class>FontListView</class>
   <extends>QTableView</extends>
   <header>fontlistview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
